<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1390</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>1191</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="mainLayout">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSave">
         <property name="text">
          <string>Save packet</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonPool">
         <property name="text">
          <string>Open pool</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QTabWidget" name="mainTabWidget">
       <property name="currentIndex">
        <number>4</number>
       </property>
       <widget class="QWidget" name="tabData">
        <attribute name="title">
         <string>Page</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>100</y>
           <width>451</width>
           <height>131</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="mainLayoutData">
          <item row="1" column="0">
           <widget class="QTextEdit" name="textEditData"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>Data:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tabTcp">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>70</y>
           <width>865</width>
           <height>276</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="mainLayoutTcp">
          <item>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="2" column="3">
             <widget class="QLineEdit" name="lineEdit_3"/>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_2"/>
            </item>
            <item row="0" column="3">
             <widget class="QLineEdit" name="lineEdit_10"/>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_9"/>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="lineEdit_12"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Acknowledgement:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Sequense:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Destination port (1-65535):</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Source port (1-65535):</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_11">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Window:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="checkBox_8">
              <property name="text">
               <string>Check sum:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QLineEdit" name="lineEdit_4"/>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Urgent:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QCheckBox" name="checkBox_17">
              <property name="text">
               <string>Header len (1-15):</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="checkBox_7">
              <property name="text">
               <string>URG</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>ACK</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_3">
              <property name="text">
               <string>PSH</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_6">
              <property name="text">
               <string>RST</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_4">
              <property name="text">
               <string>SYN</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_5">
              <property name="text">
               <string>FIN</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tabUdp">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>30</y>
           <width>865</width>
           <height>311</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="mainLayoutUdp">
          <item>
           <layout class="QGridLayout" name="layoutUdp_2">
            <item row="1" column="3">
             <widget class="QLineEdit" name="lineEdit_8">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Source port (1-65535):</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLineEdit" name="lineEdit_7"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_5"/>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_6"/>
            </item>
            <item row="1" column="2">
             <widget class="QCheckBox" name="checkBox_2">
              <property name="text">
               <string>Check sum</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Destination port (1-65535):</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="checkBox_16">
              <property name="text">
               <string>Length:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tabIcmp">
        <attribute name="title">
         <string>Page</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>50</y>
           <width>565</width>
           <height>291</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="mainLayoutIcmp">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_53">
              <property name="text">
               <string>ICMP type (0-255):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_57"/>
            </item>
            <item>
             <widget class="QLabel" name="label_54">
              <property name="text">
               <string>Code (0-255):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_58"/>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_31">
              <property name="text">
               <string>Check sum:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_59">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QLabel" name="label_55">
              <property name="text">
               <string>Sequense (0-65535):</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_60"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_56">
              <property name="text">
               <string>ID (0-65535):</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_61"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tabIp">
        <attribute name="title">
         <string>Page</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_6">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>100</y>
           <width>729</width>
           <height>284</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="mainLayoutIp">
          <item row="1" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="3" column="1">
                 <widget class="QLineEdit" name="lineEdit_23"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_20">
                  <property name="text">
                   <string>Header length (5-15): </string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_19">
                  <property name="text">
                   <string>Ip version (1-15):</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="lineEdit_22">
                  <property name="text">
                   <string>4</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLineEdit" name="lineEdit_27"/>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="checkBox_10">
                    <property name="text">
                     <string>D</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="checkBox_11">
                    <property name="text">
                     <string>T</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="checkBox_12">
                    <property name="text">
                     <string>R</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="checkBox_13">
                    <property name="text">
                     <string>M</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_24">
                  <property name="text">
                   <string>Type of service (0-7):</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="lineEdit_21"/>
                </item>
                <item row="3" column="0">
                 <widget class="QCheckBox" name="checkBox_18">
                  <property name="text">
                   <string>Total length (0-65535):</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="1">
                 <widget class="QLineEdit" name="lineEdit_25"/>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="lineEdit_24"/>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_22">
                  <property name="text">
                   <string>Identefication (0-65535):</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_23">
                  <property name="text">
                   <string>Time to live (0-255):</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QCheckBox" name="checkBox_9">
                  <property name="text">
                   <string>Header checksum (0-65535):</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_25">
                  <property name="text">
                   <string>Fragment offset (0-8192):</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="lineEdit_26">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item>
                   <widget class="QLineEdit" name="lineEdit_28"/>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="checkBox_14">
                    <property name="text">
                     <string>DF</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="checkBox_15">
                    <property name="text">
                     <string>MF</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="7">
               <widget class="QLineEdit" name="lineEdit_20">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>.</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>.</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLineEdit" name="lineEdit_14">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>To ip:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLineEdit" name="lineEdit_18">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>.</string>
                </property>
               </widget>
              </item>
              <item row="0" column="7">
               <widget class="QLineEdit" name="lineEdit_16">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="1" column="5">
               <widget class="QLineEdit" name="lineEdit_19">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="0" column="5">
               <widget class="QLineEdit" name="lineEdit_15">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="1" column="6">
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>.</string>
                </property>
               </widget>
              </item>
              <item row="0" column="6">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>.</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>.</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>From ip:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_17">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_13">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="0" column="8">
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>Auto</string>
                </property>
               </widget>
              </item>
              <item row="1" column="8">
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>Auto</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>100</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_26">
                <property name="text">
                 <string>Protocol:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_2">
                <property name="text">
                 <string>TCP</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton">
                <property name="text">
                 <string>UDP</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_3">
                <property name="text">
                 <string>ICMP</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1390</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
